### ISSUE TITLE
FNX2-13657 ⁃ Add a scrollbar indicator to the tabs tray so users know where their scroll position is in the tabs tray

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#11873]
Please add a  scrollbar so users know where exactly their position in the in the tabs tray is, for example if they're in the middle, the end or the starting position of the tabs stack. 

Right now it is hard to know the position if there're lots of tabs, let's say 50, unless the user scrolls to the end or the start of the tabs stack.

I suggested this in the issue https://github.com/mozilla-mobile/fenix/issues/11836 in a [comment](https://github.com/mozilla-mobile/fenix/issues/11836#issuecomment-648550369).

**Comment by topotropic** [comment]
Makes sense but I'm wondering why I already see a scrollbar in the tab tray in nightly from monday July 6th (build #21880609)

**Comment by SS1113** [comment]
**topotropic** you are correct sometimes I do see a scrollbar but other times I don't see any no matter how many times I restart the browser. If I have to guess, I'd say in some Nightly updates it appears and in some it doesn't. I have been thinking about filing an issue for this but I'm was not sure if it's a bug, intended behaviour, or something random. I don't know.

In fact while filing this bug I did see a scrollbar but it was after filing this issue and at the same time on the beta version it wasn't appearing. But after I updated the Nightly version next day it disappeared.
